src/features/runtime-checks.js

```diff
@@ -106,6 +106,8 @@ class DDGRuntimeChecks extends HTMLElement {
                     },
                     set (value) {
                         if (shouldFilterKey(this.#tagName, 'property', prop)) return
+                        // Ignore any properties that start with 'on' as they are only relevant once and already handled
+                        if (prop.startsWith('on')) return
```

Comment:
just a sanity check, are _all_ `on*` props problematic, or are there any that are not to do with events? I don't think there are, but wanted to ask anyway :)